kafka-connect-oura-source/src/main/java/org/radarbase/connect/rest/oura/user/OuraServiceUserRepository.kt

```diff
@@ -235,7 +235,7 @@ class OuraServiceUserRepository : OuraUserRepository() {
 
     override fun hasPendingUpdates(): Boolean =
         runBlocking(Dispatchers.Default) {
-            userCache.isStale()
+            userCache.isStale(1.hours)
```

Comment:
Does this mean it will refresh token every 1 hr instead of 5 mins? Should we do when the token expires instead of a fixed time?